Robot_Engine/Convert/ToRobot/Release.cs

```diff
@@ -50,6 +50,21 @@ public static void RobotRelease(IRobotBarEndReleaseData rData, Constraint6DOF bh
 
         /***************************************************/
 
+        public static void RobotLinearRelease(IRobotLinearReleaseData rData, Constraint4DOF bhomRelease)
```

Comment:
Think we will want to get all of these kinds of implicit comcall methods moved to the Adapter. Would not block the PR for that reason, as we generally have not followed this principle so far, but would aim to keep this in the Create method instead. WIll be significantly simpler to find as well.

Reply:
I might leave this for a later refactor... otherwise may be more confusion when moving parts of the code. Agree though, needs to go in the adapter - quite a bit of cleanup required! I'm not sure it should be kept in the create method as it's called by update too.